music_player.py

```
@@ -15,8 +17,12 @@
 
 arguments = sys.argv[2:]
 
+
 if arguments:
   for argument in arguments:
-    print(f'Playing', argument)
+    if argument in valid_files:
+        print(f'Playing', argument)
+    else:
+        print(f"can't be found or accessed{argument}")
```

You should raise an exception, and this process should happen before starting to "play" the files